src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.module.scss

```diff
@@ -18,13 +18,24 @@
   margin-bottom: 1rem;
 }
 
+.keyStatTile:first-child:nth-last-child(4),
+.keyStatTile:first-child:nth-last-child(4) ~ .keyStatTile {
```

Comment:
Could simplify this by putting this in the `.keyStatTile` block and using `&`. `&` is basically just a placeholder for the current   wrapping block's selector i.e. `.keyStatTile`.

Alternatively, I'd at least put it under `.keyStatTile` so that the ordering of blocks makes sense.